BuilderES5.js

```diff
@@ -0,0 +1,15 @@
+export default function BuilderES5 (initialValue) {
+  this._currentResult = initialValue
+}
+
+BuilderES5.prototype.plus = function (...values) {
+  for (const value of values) {
```

Comment:
The for ... of loop appeared only in the es6

Reply:
replaced with the regular for loop